uui/components/overlays/Alert.tsx

```diff
@@ -11,15 +11,24 @@ import { ReactComponent as CrossIcon } from '../../icons/navigation-close-24.svg
 import css from './Alert.module.scss';
 
 type notificationAction = {
+    /*
+    * Defines notification name.
```

Comment:
It's not an notification name, it's name of the action which will be action caption

Reply:
fixed